proxy/src/coco.rs

```diff
@@ -517,185 +48,9 @@ impl entity::Resolver<user::User<entity::Draft>> for FakeUserResolver {
 // TODO(xla): Transform into Peer::create_user.
 /// Constructs a fake user to be used as an owner of projects until we have more permanent key and
 /// user management.
-pub async fn fake_owner(peer: &Peer) -> User {
-    let key = peer
-        .with_api(|api| api.key().clone())
-        .expect("failed to get key");
+pub async fn fake_owner(key: keys::SecretKey) -> User {
```

Comment:
Why is this one left in here? Wouldn't it be better to have it with the peer or config module?

Reply:
I think it should be removed altogether. I'm working on the `rad/self` update and we need the certifier of a `Project` to exist in the monorepo. So that means our first order of business is to create the `User` in the monorepo. Meaning we should change all the calls to `fake_owner` to `peer.init_user(key, "cloudhead")`.

Long story short, I suggest we leave it for now and it'll be deleted in a future PR.